src/components/ViewModal.js

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+/*import ReactDOM from "react-dom";*/
+import Modal from 'react-bootstrap/Modal';
+
+import styles from './ViewModal.module.css';
+
+function ViewModal({ show, onClose, image, title, time, location, description, mem_limit, mem_array}) {
```

Comment:
potentially is there a way we can use a useContext or Jotai to prevent passing all these props from ViewModal to Modal?